TypeScript.tmLanguage

```diff
@@ -3172,6 +3172,11 @@
           <key>2</key>
           <dict>
             <key>name</key>
+            <string>keyword.control.loop.ts</string>
```

Comment:
I think this should be keyword.control.flow because that's how rest of the await is scoped.